release-notes/5.2/5.2.0-preview2.md

```diff
@@ -0,0 +1,84 @@
+# Release Notes
+
+## [Preview Release 5.2.0-preview2.x] - 2023-06-08
+
+This update brings the below changes over the previous release:
+
+### Added
+
+- Added SQL Types compatibility with **.NET 8**. [1934](https://github.com/dotnet/SqlClient/pull/1934)
+- Added support for **.NET 7.0**. [#1981](https://github.com/dotnet/SqlClient/pull/1981)
+- Added new property 'RowsCopied64' to SqlBulkCopy. [#2004](https://github.com/dotnet/SqlClient/pull/2004)
```

Comment:
Ticks seem wrong here

Reply:
is your concern addressed by removing the tick from SqlBulkCopy?

Reply:
He's saying they are the wrong ticks/quote character. Should be backquote instead of forward quote. They have been changed since the original commit, so this is resolved.